Corona-Warn-App/src/main/java/de/rki/coronawarnapp/worker/BackgroundWorkSchedulerBase.kt

```diff
@@ -0,0 +1,9 @@
+package de.rki.coronawarnapp.worker
+
+import de.rki.coronawarnapp.submission.SubmissionSettings
+import javax.inject.Inject
+
+@Suppress("UnnecessaryAbstractClass")
+abstract class BackgroundWorkSchedulerBase {
+    @Inject internal lateinit var submissionSettings: dagger.Lazy<SubmissionSettings>
```

Comment:
Does this need to be `lazy`?

Reply:
probably not

Reply:
i stole it from the other base impl where it was also lazy